YvesIraguha/rhinos-app
# ft(languages): support multiple languages

#### What does this PR do?

- Support multiple languages

#### Description of Task to be completed? 

- Configure localization
- Customize messages to be translated into different languages
- Add English as default language
- Add dropdown to choose language

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch **`feat/support-multiple-language`**

- Then run **`yarn install`** to install all dependencies

- Run **`yarn start`** to start the server

- In your default browser navigate to  **`http://localhost:3000/`**

- In dropdown choose any supported language

> _You should see welcome message translated to whatever language you choose_

>  _You should also see that the languages displayed in dropdown also translated to the current language_


#### Any background context you want to provide?

- To get custom message from the backend you need to have to run server on this PR [chore/support-multiple-language](https://github.com/YvesIraguha/lost-and-found-api/pull/48)

**N.B** :  - To get messages from the backend you should pass `Accept-Language` as key in headers and import function in helpers and set as the value of `Accept-Language` in headers

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2021-03-07 at 11 52 05" src="https://user-images.githubusercontent.com/54619678/110236656-ab632200-7f3f-11eb-8b50-00d1479ddf1c.png">



#### Questions:

- N/A
